src/reducers/deeplink.js

```diff
@@ -1,13 +1,16 @@
 import {
   SET_DEEPLINK_DATA,
-    SET_DEEPLINK_URL
+    SET_DEEPLINK_URL,
+    SET_DEEPLINK_DATA_EXTRAPARAMS
   } from "../utils/constants/storeType";
   
   export const deeplink = (
     state = {
       url: null,
       data: {},
-      id: null
+      id: null,
+      fromService: null,
+      extraParams: null
```

Comment:
Also, now that you've added an extra field to the deeplink reducer, you need to edit resetDeeplinkData to clear out "extraParams" (or "passthrough" after the change) so that data isn't left there when another deeplink is scanned.